docs/getting-started/istio/index.md

```diff
@@ -30,38 +30,79 @@ spec:
       stableService: rollouts-demo-stable
       trafficRouting:
         istio:
-          virtualService:
-            # Reference to a VirtualService which the controller updates with canary weights
-            name: rollouts-demo-vsvc
+          virtualServices:
+          # One or more virtualServices can be configured
+          # Reference to a VirtualService which the controller updates with canary weights.
+          - name: rollouts-demo-vsvc1
             routes:
             - primary # optional if there is a single route in VirtualService, required otherwise
+          # Reference to a VirtualService which the controller updates with canary weights
+          - name: rollouts-demo-vsvc2
+            routes:
+            - secondary # optional if there is a single route in VirtualService, required otherwise
 ...
 ```
 
-The VirtualService and route referenced in `trafficRouting.istio.virtualService` is required
-to have a HTTP route which splits between the stable and canary Services, referenced in the rollout.
+The VirtualService and route referenced in either `trafficRouting.istio.virtualService` or
+`trafficRouting.istio.virtualServices`. `trafficRouting.istio.virtualServices` helps in adding
+one or more virtualServices unlike `trafficRouting.istio.virtualService` where only single virtualService can be added.
+This is required to have a HTTP route which splits between the stable and canary Services, referenced in the rollout.
+In this example, primary route splits the http route with port number 15372 between stable and canary services;
+secondary route splits the http route with port number 15373 between stable and canary services.
 In this guide, those Services are named: `rollouts-demo-stable` and `rollouts-demo-canary` 
 respectively. The weight values for these services used should be initially set to 100% stable, 
 and 0% on the canary. During an update, these values will be modified by the controller.
+If there are multiple VirtualService then weight values for stable and canary service of each VirtualService
+will be modified by the controller.
```

Comment:
how about "... each VirtualService will be modified by the controller." -> "each VirtualService will be modified by the controller simultaneously" ?